Splitting Vector Databases into individual cookbooks

colin-openai opened the pull request:
This is the first stab at splitting these out individually - there is a common description for all, so it may be worth taking this out and putting it in the README.md instead.

Two still to be finalised:
- Milvus
- Zilliz

Four that do not have a Getting Started notebook but do have a folder:
- AnalyticDB
- Hologres
- Kusto
- SingleStoreDB

ted-at-openai commented:
Awesome; thank you. I think this makes it way more readable.

Suggestion: do you think it makes sense to stick all of the `Using_X_for_embeddings_search.ipynb` into the folders for each vector DB provider?

ted-at-openai commented on examples/vector_databases/README.md:
```diff
@@ -0,0 +1,21 @@
+# Vector Databases
+
+This section of the cookbook showcases many of the vector databases available to support your semantic search use cases. 
```
suggestion: 'the cookbook' -> 'the OpenAI Cookbook'

ted-at-openai commented on examples/vector_databases/README.md:
```diff
@@ -0,0 +1,21 @@
+
+This section of the cookbook showcases many of the vector databases available to support your semantic search use cases. 
+
+Vector databases can be a great accompaniment for knowledge retrieval applications, which reduce hallucinations by providing the LLM with the relevant context to answer their questions.
```
suggestion: 'their' -> ''

ted-at-openai commented:
^is the idea that these are all OpenAI-written whereas the ones in the folder are third-party-written?

(my instinct from a user POV is to put them all together in single folders to make it easier for people to find everything)

colin-openai commented:
@ted-at-openai got it, moved them all into the named provider folders and updated the wording in the Readme. Can add more detail to the Readme but otherwise I think we're good to go.

ted-at-openai commented:
Whoops, I merged an update by Zilliz which is blocking merging. Mind resolving? Otherwise if you think it's good to go, feel free to merge whenever you like.

colin-openai commented:
@ted-at-openai solved merge issue, merging results
